third-party/jemalloc/README

```diff
@@ -0,0 +1,11 @@
+==========================
+jemalloc README for Chapel
+==========================
+
+This copy of jemalloc is being released with Chapel for convenience and
+uses a snapshot from:
+
+  http://www.canonware.com/jemalloc/
```

Comment:
I get "Server not found" for this web address.  (Tried to go there to answer my question above.)